Configure

```diff
@@ -5356,6 +5357,21 @@ none) libs=' ';;
 *) libs="$ans";;
 esac
 
+: see if OpenSSL is available
+ossl_p_c='/home/linuxbrew/.linuxbrew/opt/openssl'
```

Comment:
What is `ossl_p_c` intended to be a mnemonic for?

Reply:
I had it spelled out as openssl_prefix_candidates but the lines got awfully long but it is a poor abbreviation and maybe I should just go back to the long version.